### PR TITLE
Add Intel 'Rooks County' NUC M15 support

### PR DESCRIPTION
**Pull request by eugenhu**
Discussed in https://github.com/thesofproject/linux/issues/4088.

### CODE REVIEW

**Comment by sofci** [comment]
Can one of the admins verify this patch?
> reply **test&nbsp;this&nbsp;please** to run this test once

File: drivers/soundwire/dmi-quirks.c
Comment (plbossart) [inline drivers/soundwire/dmi-quirks.c:92]: Commit message: you need to add the prefix 'soundwire: dmi-quirks:'

and without your Signed-off-by tag no one will look at the code

File: sound/soc/intel/boards/sof_sdw.c
Comment (plbossart) [inline sound/soc/intel/boards/sof_sdw.c:226]: This needs to be in a different patch with the prefix
ASoC: Intel: sof_sdw: 

and with a Signed-off-by tag as well.

Different subsystem and maintainer...

File: sound/soc/intel/common/soc-acpi-intel-adl-match.c
Comment (plbossart) [inline sound/soc/intel/common/soc-acpi-intel-adl-match.c:634]: and this one needs the prefix ASoC: Intel: soc-acpi: 

and your Signed-off-by.

**Review by plbossart — CHANGES_REQUESTED** [review]
looks good but formatting/signoff required. Thanks!

File: drivers/soundwire/dmi-quirks.c
Comment (bardliao) [inline drivers/soundwire/dmi-quirks.c:76]: It is better to use `Link` instead of `BugLink:` in commit message

Comment (eugenhu) [reply drivers/soundwire/dmi-quirks.c:76]: I see, I've updated the commit messages.

**Review by bardliao — APPROVED** [review]

**Review by ujfalusi — APPROVED** [review]

**Comment by plbossart** [comment]
test this please

**Review by plbossart — APPROVED** [review]

**Comment by plbossart** [comment]
SOFCI TEST

**Comment by plbossart** [comment]
for some reason I can't merge this branch, GitHub tells me:

Merge attempt failed
this branch can't be rebased

yay 2023

**Comment by bardliao** [comment]
SOFCI TEST

**Comment by plbossart** [comment]
GitHub is still completely stupidly useless,  trying with the same code in a different PR https://github.com/thesofproject/linux/pull/4126

**Comment by plbossart** [comment]
merged https://github.com/thesofproject/linux/pull/4126 instead. Thanks @eugenhu for your work.

**Pull request closed without merging**
